environment-dev.yml

```diff
@@ -2,54 +2,44 @@ name: pyjanitor-dev
 channels:
   - conda-forge
 dependencies:
-  - python=3.9
-  - biopython
-  - black=22.1.0
-  - bump2version=1.0.1
-  - cairo
-  - conda
-  - darglint
-  - doc8
-  - flake8
-  - hypothesis
-  - interrogate
-  - ipykernel
```

Comment:
drop ipykernel there should 
```diff
       - name: Build docs
         run: |
           conda activate pyjanitor-dev
-          python -m ipykernel install --user --name pyjanitor-dev
           pip install -e .
           mkdocs build
```
to fix https://github.com/pyjanitor-devs/pyjanitor/runs/7332617146?check_suite_focus=true

Reply:
When I removed this line (cfeaad7). the doc-preview ci still failed.
Due to `mkdocstring` which stays in a lower version.
So it is time to update it. With these new changes, I'm going to open a new PR #1138.
After that pr is merged, I'll reset the branch of this PR to cfeaad7 to keep it clean.